<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountAssistantClass</class>
 <widget class="QMainWindow" name="AccountAssistantClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AccountAssistant</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="pages">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_main">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lb_searchAccount">
            <property name="text">
             <string>账户检索</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="combo_searchRule">
            <item>
             <property name="text">
              <string>检索并输出全部</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>依据自定义名称</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>依据用户名</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>依据账户分组</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="combo_accountGroup">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ledit_searchInput"/>
          </item>
          <item>
           <widget class="QPushButton" name="btn_doSearch">
            <property name="text">
             <string>检索</string>
            </property>
            <property name="icon">
             <iconset resource="resources/resources.qrc">
              <normaloff>:/icon/search.svg</normaloff>:/icon/search.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lb_resultCount_tip">
            <property name="text">
             <string>结果数量:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lb_resultCount_value">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lb_lastSyncTime_tip">
            <property name="text">
             <string>上次同步时间:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lb_lastSyncTime_value">
            <property name="text">
             <string>2023-10-23 12:52:00</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_remoteSync">
            <property name="text">
             <string>远程同步</string>
            </property>
            <property name="icon">
             <iconset resource="resources/resources.qrc">
              <normaloff>:/icon/sync.svg</normaloff>:/icon/sync.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_newArchive">
            <property name="text">
             <string>新建存档</string>
            </property>
            <property name="icon">
             <iconset resource="resources/resources.qrc">
              <normaloff>:/icon/add.svg</normaloff>:/icon/add.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="table_resultShow">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="tabKeyNavigation">
           <bool>false</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>80</number>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>自定义名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>用户名</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>密码</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>-</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>-</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>分组</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>-</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>-</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>id</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_setting">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QScrollArea" name="scroll_settingOptionsContainer">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>444</width>
             <height>60</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="layout_settingPageOptions_1">
              <item>
               <widget class="QLabel" name="lb_windowCLoseAction">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>关闭按钮行为</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="combo_windowCloseAction">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <item>
                 <property name="text">
                  <string>最小化</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>退出程序</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="check_hideMainWindowWhenStart">
                <property name="layoutDirection">
                 <enum>Qt::RightToLeft</enum>
                </property>
                <property name="text">
                 <string>启动时隐藏主窗口</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="check_showSystemMessageWhenStart">
                <property name="layoutDirection">
                 <enum>Qt::RightToLeft</enum>
                </property>
                <property name="text">
                 <string>启动时显示系统提示消息</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="layout_settingPageOptions_2">
              <item>
               <widget class="QLabel" name="lb_clipboardWriteContent">
                <property name="text">
                 <string>复制账户时向剪切板内写入的内容</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="combo_clipboardWriteContent">
                <item>
                 <property name="text">
                  <string>用户名</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>密码</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>全部</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lb_clipboardWriteMode">
                <property name="text">
                 <string>写入模式</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="combo_clipboardWriteMode">
                <item>
                 <property name="text">
                  <string>合并后写入</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>监听模式写入</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="tbtn_clipboardWriteHelp">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;当写入内容为全部时提供两种写入模式:&lt;/p&gt;&lt;p&gt;① 合并后写入:&lt;/p&gt;&lt;p&gt;该模式下复制账户时会将用户名与密码用空格连接后一次性写入剪切板&lt;/p&gt;&lt;p&gt;② 监听模式写入:&lt;/p&gt;&lt;p&gt;该模式下复制账户时会首先将用户名写入剪切板, 同时监听&amp;quot;Ctrl + V&amp;quot;快捷键,&lt;/p&gt;&lt;p&gt;当按下此快捷键后自动将密码写入剪切板&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="toolTipDuration">
                 <number>60000</number>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset resource="resources/resources.qrc">
                  <normaloff>:/icon/help.svg</normaloff>:/icon/help.svg</iconset>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layout_settingPageButtonBox">
          <item>
           <spacer name="hs_1">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_settingReject">
            <property name="text">
             <string>取消</string>
            </property>
            <property name="icon">
             <iconset resource="resources/resources.qrc">
              <normaloff>:/icon/close.svg</normaloff>:/icon/close.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_settingConfirm">
            <property name="text">
             <string>确认</string>
            </property>
            <property name="icon">
             <iconset resource="resources/resources.qrc">
              <normaloff>:/icon/save.svg</normaloff>:/icon/save.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_settingApply">
            <property name="text">
             <string>应用</string>
            </property>
            <property name="icon">
             <iconset resource="resources/resources.qrc">
              <normaloff>:/icon/done.svg</normaloff>:/icon/done.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_option">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="action_option_setting"/>
    <addaction name="action_option_accountGroupSetting"/>
    <addaction name="separator"/>
    <addaction name="action_option_quit"/>
   </widget>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_file_import"/>
    <addaction name="action_file_export"/>
   </widget>
   <widget class="QMenu" name="menu_safe">
    <property name="title">
     <string>安全</string>
    </property>
    <addaction name="action_security_setting"/>
   </widget>
   <addaction name="menu_option"/>
   <addaction name="menu_file"/>
   <addaction name="menu_safe"/>
  </widget>
  <action name="action_option_setting">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icon/settings.svg</normaloff>:/icon/settings.svg</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="toolTip">
    <string>自定义设置</string>
   </property>
  </action>
  <action name="action_option_quit">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icon/power.svg</normaloff>:/icon/power.svg</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_file_import">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icon/upload.svg</normaloff>:/icon/upload.svg</iconset>
   </property>
   <property name="text">
    <string>导入数据</string>
   </property>
  </action>
  <action name="action_file_export">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icon/download.svg</normaloff>:/icon/download.svg</iconset>
   </property>
   <property name="text">
    <string>导出数据</string>
   </property>
  </action>
  <action name="action_security_setting">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icon/lock.svg</normaloff>:/icon/lock.svg</iconset>
   </property>
   <property name="text">
    <string>安全设置</string>
   </property>
  </action>
  <action name="action_option_accountGroupSetting">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icon/label.svg</normaloff>:/icon/label.svg</iconset>
   </property>
   <property name="text">
    <string>分组管理</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
